wondrous-app/components/Common/Task/Task.tsx

```diff
@@ -246,7 +228,7 @@ export function Task(props) {
     });
   };
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
Same q here

Reply:
Im not sure now if it even need to be wrapped in effect hook, maybe it's better to be just with useMemo or something, because there is no logic expect for setUserList, like:
```
const usersList = useMemo(() => {
  return assigneeUsername 
   ? [{
     id: assigneeId,
     ...
    }]
    : []
}, [assigneeUsername, assigneeId]);
```

Reply:
agree with @lisovskyR , better to rewrite and use `useMemo` hook

Reply:
Ok, will do it

Reply:
done